### DISCUSSION TITLE
Joining GSoC

### DISCUSSION THREAD

**Discussion by VaibhavDS19** [Rust-GCC/gccrs#356] [Q&A]
I am interested in GSoC and wanted to know how to specifically work on gccrs. I have no actual experience in writing compilers but I do know some basic level of Rust. Is there a way I can contribute to this? Where do I apply? It doesnt necessarily have to be for GSoC either, I just want to contribute. Kindly guide me here.

**Comment by tschwinge** [comment]
See <https://rust-gcc.github.io/> for general information, <https://github.com/Rust-GCC/gccrs/wiki/Google-Summer-of-Code> for GSoC.  Most of the GCC/Rust discussion happens in <https://gcc-rust.zulipchat.com/>.

First steps usually are download, build, experiment, run into something broken, try fixing it, ask questions, submit fix -- or similar.  :-)
